File: Q09/main.cpp
#include<iostream>
using namespace std;

int main(){
    int row,col;
    cin>>row>>col;

    int arr[row][col];

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            cin>>arr[i][j];
        }
    }

    for(int i=0;i<row;i++){
        int max = arr[i][0];
        for(int j=0;j<col;j++){
            if(arr[i][j]>max){
                max = arr[i][j];
            }
        }
        cout<<max<<" ";
    }
    return 0;
}

File: Q09/main.py
row = int(input())
col = int(input())
arr=[]

for i in range(row):
    arr.append(list(map(int,input().split()))[:col])


for i in range(row):
    max = arr[i][0]
    for j in range(col):
        if arr[i][j]>max:
            max=arr[i][j]
    print(max,end=" ")